tap_github/streams.py

```diff
@@ -904,3 +907,171 @@ def http_headers(self) -> dict:
+    ).to_dict()
+
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        """Parse the response and return an iterator of result rows."""
+        yield from extract_jsonpath(self.records_jsonpath, input=response.json())
+
+    def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
+        """Return a child context object from the record and optional provided context.
+
+        By default, will return context if provided and otherwise the record dict.
+        Developers may override this behavior to send specific information to child
+        streams for context.
+        """
+        return {
+            "org": context["org"] if context else None,
+            "repo": context["repo"] if context else None,
+            "run_id": record["id"],
+        }
+
+
+class WorkflowRunJobsStream(GitHubStream):
+    """Defines 'workflow_run_jobs' stream."""
+
+    MAX_PER_PAGE = 100
+
+    name = "workflow_run_jobs"
+    path = "/repos/{org}/{repo}/actions/runs/{run_id}/jobs"
+    primary_keys = ["id"]
+    parent_stream_type = WorkflowRunsStream
+    ignore_parent_replication_key = False
+    state_partitioning_keys = ["repo", "org", "run_id"]
+    records_jsonpath = "$.jobs[*]"
+
+    schema = th.PropertiesList(
+        # Parent keys
+        th.Property("repo", th.StringType),
+        th.Property("org", th.StringType),
+        # PR keys
+        th.Property("id", th.IntegerType),
+        th.Property("run_id", th.IntegerType),
+        th.Property("run_url", th.StringType),
+        th.Property("node_id", th.StringType),
+        th.Property("head_sha", th.StringType),
+        th.Property("url", th.StringType),
+        th.Property("html_url", th.StringType),
+        th.Property("status", th.StringType),
+        th.Property("conclusion", th.StringType),
+        th.Property("started_at", th.DateTimeType),
+        th.Property("completed_at", th.DateTimeType),
+        th.Property("name", th.StringType),
+        th.Property(
+            "steps",
+            th.ArrayType(
+                th.ObjectType(
+                    th.Property("name", th.StringType),
+                    th.Property("status", th.StringType),
+                    th.Property("conclusion", th.StringType),
+                    th.Property("number", th.IntegerType),
+                    th.Property("started_at", th.DateTimeType),
+                    th.Property("completed_at", th.DateTimeType),
+                )
+            ),
+        ),
+        th.Property("check_run_url", th.StringType),
+        th.Property("labels", th.ArrayType(th.StringType)),
+        th.Property("runner_id", th.IntegerType),
+        th.Property("runner_name", th.StringType),
+        th.Property("runner_group_id", th.IntegerType),
+        th.Property("runner_group_name", th.StringType),
+    ).to_dict()
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._schema_emitted = False
```

Comment:
@aaronsteers I also made schema messages a bit quieter for jobs. It was outputting one every time a new parent workflow run was synced, and some targets flush records when they receive a new schema for an existing table, which would cause the target to write really small batches and affect performance.

Maybe this makes sense as a builtin option in the SDK?